packages/web-runtime/src/pages/account.vue

```diff
@@ -288,33 +304,19 @@ export default defineComponent({
 
     const updateSelectedLanguage = async (option: LanguageOption) => {
       try {
-        const value = await saveValue({
-          identifier: 'language',
-          valueOptions: { listValue: { values: [{ stringValue: option.value }] } }
-        })
+        await clientService.graphAuthenticated.users.editUser(unref(graphUser).id, {
+          preferredLanguage: option.value
+        } as any)
         selectedLanguageValue.value = option
         setCurrentLanguage({
           language,
-          languageSetting: {
-            identifier: {
-              extension: 'ocis-accounts',
-              bundle: 'profile',
-              setting: 'language'
-            },
-            value
-          }
+          languageSetting: option.value
         })
         if (unref(sseEnabled)) {
           ;(clientService.sseAuthenticated as SSEAdapter).updateLanguage(language.current)
         }
-        if (unref(personalSpace)) {
-          // update personal space name with new translation
-          store.commit('runtime/spaces/UPDATE_SPACE_FIELD', {
-            id: unref(personalSpace).id,
-            field: 'name',
-            value: unref(spacesEnabled) ? $gettext('Personal') : $gettext('All files')
-          })
-        }
```

Comment:
Why remove that?

Reply:
Thats oc10 code.

Reply:
Uhm, weird code path, could you please check if the latest commit is suffcient.